src/services/utilities.ts

```diff
@@ -1435,14 +1435,25 @@ namespace ts {
 
     const displayPartWriter = getDisplayPartWriter();
     function getDisplayPartWriter(): DisplayPartsSymbolWriter {
+        const maximumApproximateLength = 2000;

```

Comment:
that seems a lot.. not sure what is a good number.. but 100 seems slightly too short.. so may be something in the range of 150 - 200..

Reply:
`240` because that's 3 80 character lines?